app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -271,7 +271,7 @@
               <div style="font-size: 16px; margin: 8px 0 12px 0;">  <%= raw t("analytics.result_from_last_visit_in") %> <%= cohort[:results_in] %></div>
               <table class="cohort-bars">
                   <tr>
-                    <% if cohort[:registered].zero? %>
+                    <% if cohort[:registered].blank? %>
```

Comment:
This fixes the bug seen in https://app.shortcut.com/simpledotorg/story/6069/actionview-template-error-undefined-method-zero-for-nil-nilclass ch6069